isis/src/control/apps/cnetextract/cnetextract.h

```diff
@@ -0,0 +1,38 @@
+#ifndef cnetextract_h
+#define cnetextract_h
+
+#include <set>
+#include <sstream>
+
+#include <QMap>
+#include <QSet>
+#include <QVector>
+
+#include "Angle.h"
+#include "Camera.h"
+#include "ControlMeasure.h"
+#include "ControlNet.h"
+#include "ControlPoint.h"
+#include "Cube.h"
+#include "CubeManager.h"
+#include "Environment.h"
+#include "FileList.h"
+#include "IException.h"
+#include "IString.h"
+#include "Longitude.h"
+#include "Latitude.h"
+#include "TProjection.h"
+#include "Progress.h"
+#include "ProjectionFactory.h"
+#include "Pvl.h"
+#include "SerialNumber.h"
+#include "SurfacePoint.h"
+#include "UserInterface.h"
+
+namespace Isis {
+  void cnetextract(UserInterface &ui, Pvl *log);
+
+  void cnetextract(ControlNet outNet, UserInterface &ui, Pvl *log);
```

Comment:
missing extern, but now I'm thinking it's superfluous? I don't see a case where we plan to call the app without a header and it probably increases linking times 🤔